### DISCUSSION TITLE
Buddy System

### DISCUSSION THREAD

**Discussion by glennawatson** [dotnet-foundation/Home#45] [General Discussions]
Based on discussions from other threads, there seems to be an appeal of a buddy system.

Where a project is struggling, a buddy could help out with the project.

The buddy could be another project which has done a particular process before or helping a project set up a CI-based system for example

**Comment by jglozano** [comment]
@glennawatson I love the idea! Having projects help each other “scales” better than individual board members helping out. I say this because the board will change every two years as elections and positions end.

**Reply by glennawatson** [reply]
Can't take any credit for this one :) Just making this discussion for easy tracking of everyone.

**Comment by robmen** [comment]
If this "buddy system" is the same that I proposed elsewhere, the goal was actually to:

Buddy up every .NET Foundation project with a .NET Foundation board member (all decision-makers, really). As I've reflected on the past week, I believe that a lot of issues could have been resolved better if there was someone at the .NET Foundation that each project trusted.  Project maintainers wouldn't have felt so isolated and (ideally) a board member for each of the struggling project maintainers would jump in to ensure the communication was flowing.

Additionally, when new board members are elected, their buddy project maintainer will bring them up to speed how OSS projects work simply by communicating. Ideally, this will enable a "continuity of norms" to flow from election to election as the .NET Foundation's leadership changes.

I originally called it "adopt a project" but I thought "buddy" was a better term since I hope there is a benefit for both the project maintainer and the board member.

**Reply by jglozano** [reply]
I like the concept but again, I want to make sure it “scales” with 140+ projects and 7 board members, that’s ~20 projects per board member.

@rprouse @devlead thoughts?

**Reply by robmen** [reply]
I'm pretty low maintenance. 😉 

I wouldn't expect to chat more than twice a year, with a longer discussion when the buddy changes.

**Reply by rprouse** [reply]
I think it depends on the work involved. When I was going through the list of projects this week, a large percentage were inactive or Microsoft projects, so the 140 would likely be a much smaller number. It is definately something we can look at.

**Reply by jglozano** [reply]
@rprouse yeah, I figure you knew the numbers based on spreadsheet you were building.

**Reply by glennawatson** [reply]
So realistically, there are two separate proposals here. One is an assigned board member/buddy for projects to deal with. The other is a helper for new projects and projects the need a helping hand.
